src/thunderbots/software/util/refbox_constants.cpp

```diff
@@ -25,6 +25,18 @@ static const std::map<RefboxGameState, std::string> refbox_game_state_names = {
     {RefboxGameState::BALL_PLACEMENT_THEM, "BALL_PLACEMENT_THEM"},
     {RefboxGameState::REFBOX_GAME_STATE_COUNT, "REFBOX_GAME_STATE_COUNT"}};
 
+std::string name(const RefboxGameState& state)
```

Comment:
Doesn't look like this is covered in any unit tests?

Reply:
added tests